### ISSUE TITLE
Dashboard: Building damage

### ISSUE THREAD

**Issue by jvanulde** [OpenDRR/opendrr-api#57]
<img width="1153" alt="Screen Shot 2020-12-01 at 6 19 36 AM" src="https://user-images.githubusercontent.com/2255248/100752325-542c3f00-339d-11eb-8ed5-22be6c0acdfd.png">

**Comment by mjourneay** [comment]
Base Map | NSHL | nhsl_{site_region}_physical_exposure_all_indicators_s | E_LandUse |   |   |  
-- | -- | -- | -- | -- | -- | --
RedTag Buildings - scenario | DSRA | dsra_{scenario_id}_building_damage_damage_state_s | sDt_RedTag_b_b0,sDt_RedTag_b_r2, | dsra_{scenario_id}_building_damage_damage_state_b | (sD_Extensive_b0+sD_Complete_b0) + (sD_Extensive_r2+sD_Complete_r2) | E_LandUse, E_BldgOccG, E_BldgTypeG, E_BldgDesLev
RedTag Buildings - 500yr | PSRA | psra_{run#}_building_damage_event_based_damage_s | eDt_RedTag_b_b0, eDt_RedTag_b_r2 | psra_{run#}_building_damage_event_based_damage_b | (eD_Extensive_b0+eD_Complete_b0), (eD_Extensive_r2+eD_Complete_r2) | E_LandUse, E_BldgOccG, E_BldgTypeG, E_BldgDesLev

**Comment by jvanulde** [comment]
@drotheram I moved this into in In progress since this is what you are currently refining. Am I correct?

**Comment by drotheram** [comment]
Yes, but I think we can move all of the dashboards to 'in progress' now
